core/test/caver-js.test.ts

```diff
@@ -0,0 +1,84 @@
+import { describe, test, expect, jest } from '@jest/globals'
+import { BigNumber, ethers } from 'ethers'
+import Caver from 'caver-js'
+import { NODE_ENV } from '../src/settings'
+
+describe('Test Caver-js', function () {
+  jest.setTimeout(10000)
+  console.log('NODE_ENV:', NODE_ENV)
+
+  if (NODE_ENV == 'development') {
+    test('Send signed tx with is caver-js on Baobab', async function () {
+      const PROVIDER_URL = 'https://api.baobab.klaytn.net:8651'
+      const caver = new Caver(PROVIDER_URL)
+
+      const privateKey = '0xced84838ee2119943c7cf8eca25b5d6bf83f322cc0121504f8979f3411badd2e'
+      const address = '14fadded4f98064c3188765df46a055edebf0cf2'
+      const account1 = caver.klay.accounts.createWithAccountKeyPublic(address, privateKey)
```

Comment:
Is there a reason to add a number to account? I cannot see any other account in this test. 🤔